ani-cli

```diff
@@ -93,6 +115,32 @@ search_eps () {
 		'
 }
 
+search_for_unwatched () {	
+	# compares history with gogoanime, only shows unfinished anime
+	search_results=$*
+
+	unwatched_anime=""
+	while read -r anime_id; do
+		current_ep_number="$(search_eps "$anime_id")"
+		history_ep_number="$(sed -n -E "s/${anime_id}\t//p" "$logfile")"
+		if [ "$current_ep_number" -ge "$history_ep_number" ]
+		then
+			unwatched_anime="$unwatched_anime$anime_id\n"
+		fi
+	done <<-EOF
+	$search_results
+	EOF
+	if [ -z "$unwatched_anime" ]; then
+		die "No unwatched episodes"
+	fi
+	printf "%s""$unwatched_anime"
```

Comment:
there would be a space between the format and args here

Reply:
Nice find!